apollo-backend/src/main/java/io/logz/apollo/services/DeploymentService.java

```diff
@@ -39,6 +39,10 @@ public void logDeploymentDescription(Deployment deployment) {
             MDC.put("markers", String.format("service-name:%s", service.getName()));
             MDC.put("env", env.getAvailability());
             MDC.put("region", env.getGeoRegion());
+            MDC.put("service", service.getName());
+            MDC.put("commit", deployableVersion.getGitCommitSha());
+            MDC.put("committer", deployableVersion.getCommitterName());
```

Comment:
maybe we should use `deployment.getUserEmail()`? i am not sure "commiterName" is initialized

Reply:
@matvey-mtn The use email is used for show who asked for the deployment while "committerName" is the one who created the commit itself.

Reply:
@matvey-mtn But I see that it is not always initialized.. is Null is safe as value? if no - lets remove it . wdy Maty?

Reply:
@maorshap I think we want to see the email (or name/surname) of the person who deployed service, no?

Reply:
@matvey-mtn Yeah , also. it is can be found on the message itself.